t/test/mojo.t

```diff
@@ -5,6 +5,10 @@ use Test::Mojo;
 use Mojolicious::Lite;
 
 any '/' => {text => 'Hello Test!'};
+websocket '/websocket', => sub {
+  my ($c) = @_;
```

Comment:
We use the `= shift;` convention everywhere else.

Reply:
Done.